host/gethistory_test.go

```diff
@@ -663,13 +663,13 @@ func (s *integrationSuite) TestAdminGetWorkflowExecutionRawHistory_All() {
 	// this function poll events from history side
 	pageSize := 1
 	getHistory := func(namespace string, execution *commonpb.WorkflowExecution, firstEventID int64, nextEventID int64,
-		token []byte) (*adminservice.GetWorkflowExecutionRawHistoryResponse, error) {
```

Comment:
i would suggest delete this function (TestAdminGetWorkflowExecutionRawHistory_All) all together and file a ticket to create an integration test for v2 get raw history API

v2 api also takes in the version of start / end history event:
https://github.com/temporalio/temporal/blob/526beb5f0bffbefe92ef1ba26b9ca116c68f478e/proto/internal/temporal/server/api/adminservice/v1/request_response.proto#L114
https://github.com/temporalio/temporal/blob/526beb5f0bffbefe92ef1ba26b9ca116c68f478e/proto/internal/temporal/server/api/adminservice/v1/request_response.proto#L116

Reply:
Yeah ran into this.  Removing these tests for now.  Filed: https://github.com/temporalio/temporal/issues/745